### DISCUSSION TITLE
Referensi topik dari The Valley of Code

### DISCUSSION THREAD

**Discussion by rizafahmi** [ngobrolin/web#39] [Ideas]
Wah kalau konten ngobrolin web dibuat tulisan bisa jadi kayak ini nih https://thevalleyofcode.com/

Ada beberapa topik yang belum dibahas sepertinya menarik kalau dijadikan ide topik berikutnya.

Lama-lama kalau dikumpulin bisa jadi kurikulum 😬

cc @ivankristianto @ekafyi

**Comment by ivankristianto** [comment]
Setuju!
